cmd/podman/pod_ps.go

```diff
@@ -74,8 +75,8 @@ type podPsJSONParams struct {
 	Status             string         `json:"status"`
 	CtrsInfo           []podPsCtrInfo `json:"containerinfo,omitempty"`
 	Cgroup             string         `json:"cgroup,omitempty"`
-	InfraContainerID   string         `json:"infracontainerid,omitempty"`
-	SharedNamespaces   []string       `json:"sharednamespaces,omitempty"`
+	InfraID            string         `json:"infraid,omitempty"`
```

Comment:
Maybe InfraCtrID?

Reply:
(Mostly referring to the JSON, it's fine to call if whatever internally)

Reply:
I guess all-lower... We really ought to be camelcasing JSON tags, but nothing else in the struct does.

Reply:
Changing to camelcase SGTM

Reply:
I am also going to call it infraContainerId to be more consistent with containerInfo above. verbosity in json doesn't look as bad as template output.